### ISSUE TITLE
Implement xvfb support

### ISSUE THREAD

**Issue by bzier** [bzier/gym-mupen64plus#2]
The framework should run the emulator in an xvfb display. This should provide consistent x,y offsets, reducing the need for configuration. It should also allow for the environment to be run on a headless server without the need for an X11 display / desktop environment. Along with this, the `_render()` method should be implemented to support displaying the screen in a window when requested. [Relevant code](https://github.com/bzier/gym-mupen64plus/blob/2a692373b3495a64529094e53bbcd6ebbba70b7b/gym_mupen64plus/envs/mupen64plus_env.py#L119)

**Comment by kevinhughes27** [comment]
This would be great! I just ran into issues with the image size and offset while testing this out.

**Comment by bzier** [comment]
This is now working. The config file has been updated with a number of settings to support this feature, including a flag to turn it on or off (i.e. function in the old way by opening the emulator on your desktop). I also updated the gym-agent to work with the new `render()` functionality and fixed a bug which caused bad screenshots when running with `xvfb` (see [here](https://github.com/bzier/TensorKart/commit/1d8ae7eade99be4bc1a62c11d041f9d6aed210c0#commitcomment-21489172))
